wolfcrypt/src/rsa.c

```diff
@@ -1099,7 +1099,7 @@ static int RsaPad_PSS(const byte* input, word32 inputLen, byte* pkcsBlock,
         int saltLen, int bits, void* heap)
 {
     int   ret = 0;
-    int   hLen, i, o, maskLen, hiBits;
```

Comment:
The `o` variable is still required for `#if defined(WOLFSSL_PSS_LONG_SALT) || defined(WOLFSSL_PSS_SALT_LEN_DISCOVER)` case at line 1179. And still needed for `m[i] ^= salt[o + i];` at 1214.